src/controller/cmp.ts

```diff
@@ -1,6 +1,15 @@
 import { Request, Response } from "express";
 
-import { API_VERSION, HTTP_HOST, TECH_COOKIE_NAME, COOKIE_NAME } from "../config";
+import {
+  API_VERSION,
+  HTTP_HOST,
+  TECH_COOKIE_NAME,
+  COOKIE_NAME,
+  TRACKING_PROTOCOL,
+  TRACKING_HOST_CONSENT,
+  TRACKING_HOST_NO_CONSENT,
+  TRACKING_VERSION,
```

Comment:
This leaks implementation details of the tracking system into the CMP provider and makes the implementation hard to read. (People wonder why you suddenly need the trackign script version here)  I'd instead simply use one parameter that includes the full URL, e.g.  `SUBMIT_CONSENT_FOR_TRACKING_DEVICE_ID_URL` and set this to `http://sessions.tvpin....`